extensions/ql-vscode/src/pure/messages.ts

```diff
@@ -682,6 +707,26 @@ export interface ClearCacheResult {
   deletionMessage: string;
 }
 
+/**
+ * The result of starting a new structured log.
+ */
+export interface StartLogResult {
+  /**
+   * A user friendly message saying what happened.
+   */
+  successMessage: string;
```

Comment:
What message is written on failure? It might be too hard to change the CLI here, but should this be `completionMessage` instead to allow for the possibility of failure?

Reply:
Currently this cannot fail. I agree that calling it something like `completionMessage` to allow for the possibility of a failure in the future seems reasonable. This doesn't even _really_ need to be an object, but it seemed better to make it one, also for future extensibility.